tests/functions.php

```diff
@@ -0,0 +1,220 @@
+<?php
+
+require('../include/functions.php');
```

Comment:
I don't think you need brackets for require because it's a statement rather than a function (googled it yesterday when I wasn't sure!!)